### PR TITLE
Group events page is empty

### PR DESCRIPTION
**Pull request by edwh**
This page got broken when I created GroupPage.vue.

### CODE REVIEW

File: app/Http/Controllers/PartyController.php [hunk 1]
```diff
@@ -87,10 +87,9 @@ public function index($group_id = null)
             $upcoming_events_in_area = null;
         } else {
             $upcoming_events = Party::upcomingEvents()->where('users_groups.user', Auth::user()->id)
-                ->take(3)
                 ->get();
 
-            $past_events = Party::UsersPastEvents([auth()->id()])->paginate(10);
```
Comment (ngm) [inline app/Http/Controllers/PartyController.php:93]: Pulling back all results has a bit of a performance hit on the /party page - logged in as a user with 263 past events, it takes 7 seconds to render on my laptop, so will presumably also have the performance issue James logged for his mobile.  Not sure if there's anything we can do here right now?  But we'll need to address this soon I think.

File: app/Http/Controllers/PartyController.php [hunk 2]
```diff
@@ -87,10 +87,9 @@ public function index($group_id = null)
...
+            $past_events = Party::UsersPastEvents([auth()->id()])->get();
```
Comment (ngm) [inline app/Http/Controllers/PartyController.php:92]: Could we also sort by event_date, most recent first?

**Review by ngm — CHANGES_REQUESTED** [review]
Fixes the issue, but wondering if there's anything performance related we can do regarding the removal of the pagination?

File: app/Http/Controllers/PartyController.php (hunk 2 above)
Comment (ngm) [reply app/Http/Controllers/PartyController.php:92]: It turns out we are sorting on the query, but I think we lose this when putting items in to the store?  Have put a sort at the point of display for now, but probably a more efficient way of doing this.

Comment (edwh) [reply app/Http/Controllers/PartyController.php:92]: Will discuss at retro.

**Pull request merged**